### PR TITLE
👋 Welcome page

### PR DESCRIPTION
**Pull request by rowanc1**
This will create a landing page for our github, (for example [similar to here](https://github.com/myst-templates)) with many links to projects that aims to help orient and welcome our community.

One goal especially is to bring more visibility to discussions, the various components that are being worked on in the project, and a (very) high level description of how they fit together.

This is discussed here:
* https://github.com/executablebooks/meta/issues/835

### CODE REVIEW

File: profile/README.md
Comment (choldgraf) [inline profile/README.md:5]: The last few months have made me believe we should be more precise in our description of the EBP project and its (current) organizational structure. I think it's more accurate to say we are a _project_, funded by the Sloan Foundation, the is building open source technology. We aspire to becoming a community-driven project, but we are currently largely a single project team (e.g., everybody in [the team page](https://executablebooks.org/en/latest/team.html) is either a Principle Investigator or has been paid to work on this except for one person).

File: profile/README.md
Comment (choldgraf) [inline profile/README.md:12]: For example, I suggest that we make this more explicit and list-like. IMO our goal should be to route people to other documentation quickly that has more information. Something like:

```
See below for several ways that you can interact with this project and its team.

- [Our discussion forum](link) is where we share questions, ideas, and discuss broad issues related to the project.
	- <note: the discussion forum should then have a top-level link that explains its structure, what kind of information happens there, etc> 
- [Our team meetings](link) are a way for the project's core team and community to discuss and share important topics that benefit from real-time discussion. They are open for anybody to join.
- [Our twitter account](link) is used to share information about the project and what goes on in its community. 
```

I think that this list should be fairly minimal, and should defer to a single source of truth about things like "when are the meetings and what are their topics", or "what is in the forum".

File: profile/README.md
Comment (choldgraf) [inline profile/README.md:16]: We should put the code of conduct in a dedicated header above "Contribute to a project".

e.g.:

```
## Code of Conduct

By participating in any space stewarded by the Executable Books Project, you must follow our [Code of Conduct](link). Please review its content and the processes we have for enforcement. However, note this CoC is aspirational, as the project is transitioning to be more inclusive, and is still developing its culture and CoC procedures. 
```

File: profile/README.md
Comment (choldgraf) [inline profile/README.md]:
```suggestion
As well as many other tools for parsing, headless execution of Jupyter Notebooks ([caching](https://github.com/executablebooks/jupyter-cache) and in the [browser](https://github.com/executablebooks/thebe)), popular sphinx extensions (e.g. [copy-button](https://github.com/executablebooks/sphinx-copybutton), [sphinx-design](https://github.com/executablebooks/sphinx-design), [sphinx-thebe](https://github.com/executablebooks/sphinx-thebe)), and many [scientific templates](https://github.com/myst-templates) for writing journal articles and presentations.
```

Maybe one adverb too many :-)

**Review by choldgraf — COMMENTED** [review]
Thanks for putting this together! I added a few comments in the text. One major thing:

I'm having a hard time figuring out what kind of balance we should strike between having lots of content here, vs. using a single source of truth elsewhere (e.g. executablebooks.org) and linking to it from here. My gut feeling is that:

- we don't want to duplicate lots of content across many spaces, especially if they relate to team policy (since they'll be out of date if policy changes)
- we want a non-github space as the "central" learn-about-the-project space
			
So my feeling is that this page should be fairly minimal and largely link out to other places with more narrative and explanation. In my opinion, our `executablebooks.org` site should be the primary "source of truth" for the organization's policies and instructions for how to learn more, participate, etc. We should primarily be directing people there.

What do you think?

**Comment by rowanc1** [comment]
@choldgraf, what are your thoughts about getting an updated version of this in to help people navigate around the various repositories. Right now we have room for 6 pinned repos, which is confusing for some users (e.g. see note in https://github.com/executablebooks/mystjs/issues/213).

To address your duplicated content concerns, we could slim this PR down, but have just enough content to point to various repositories / spaces. I like the idea of the lists rather than in-text links. I think solid pointers to the website as the source of truth makes sense, and this page is only for how to navigate the many github projects we have!

**Review by choldgraf — APPROVED** [review]
Thanks for this, let's just merge this one in and iterate from there, sorry for blocking this one earlier.

**Pull request merged**
